### DISCUSSION TITLE
Modelling perennial grains - is it possible to model perennials without using Simple Leaf?

### DISCUSSION THREAD

**Discussion by pjinnes** [APSIMInitiative/ApsimX#9272] [Q&A]
I've tried to base a perennial grains model on first the wheat model and then the sorghum model, but can't find a way to get regrowth to work after the first season harvest. There is no easy way of resetting the leaf/stem/tiller numbers to an earlier vegetative stage that I can see (without modifying a lot of apsimx code). Most of the fields are read only and its not clear how resetting these affects the rest of the apsimx resource calculations. 

Using the Simple Leaf approach, e.g. Red Clover, works but this then excludes any tiller modelling. 
Any suggestions?

**Comment by HamishBrownPFR** [comment]
Hi Peter.

What you are trying to do should be possible.  The Lucerne prototype uses simple leaf and is defoliated and regrows multiple times.  Have a look at that for some ideas.  The SPRUM prototype may have something useful too.
